testapp/src/main/res/layout/settings_menu_base_activity.xml

```diff
@@ -0,0 +1,74 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:padding="@dimen/medium_16"
+  android:orientation="vertical">
+
+  <androidx.appcompat.widget.AppCompatTextView
+    android:id="@+id/textInfo"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content" />
+
+  <RelativeLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/small_8">
+
+    <androidx.appcompat.widget.AppCompatButton
+      android:id="@+id/buttonCancel"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_alignParentStart="true"
+      android:text="@string/action_cancel" />
+
+    <androidx.appcompat.widget.AppCompatButton
+      android:id="@+id/buttonSave"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_alignParentEnd="true"
+      android:text="@string/action_save"
+      tools:ignore="RelativeOverlap" />
+  </RelativeLayout>
+
+  <include layout="@layout/line_settings_horizontal" />
+
+  <androidx.appcompat.widget.SwitchCompat
+    android:id="@+id/switchTest"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:text="@string/lb_test_mode"
+    android:thumb="@drawable/bg_thumb_testmode"
+    app:track="@drawable/bg_track_testmode" />
+
+  <include layout="@layout/line_settings_horizontal" />
+
+  <com.google.android.material.textfield.TextInputLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <androidx.appcompat.widget.AppCompatEditText
+      android:id="@+id/subscription_key"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:hint="@string/hint_subscription_key"
+      android:imeOptions="actionDone"
+      android:inputType="text" />
+  </com.google.android.material.textfield.TextInputLayout>
+
+  <com.google.android.material.textfield.TextInputLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <androidx.appcompat.widget.AppCompatEditText
+      android:id="@+id/app_id"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="@dimen/small_4"
+      android:hint="@string/hint_subscription_key"
+      android:imeOptions="actionDone"
+      android:inputType="text" />
+  </com.google.android.material.textfield.TextInputLayout>
```

Comment:
`app_id` must be in top of the `subscription_key`. 
The text hint of `app_id` is wrong here.